src/shared/model/Civic.ts

```diff
@@ -25,3 +25,13 @@ export interface ICivicEntry {
     url: string;
     variants: {[name: string]: ICivicVariantData};
 };
+
+export interface ICivicGeneDataWrapper {
+    status: "pending" | "error" | "complete";
```

Comment:
We really need to have a single interface for these status values.  Why don't you make one for civic and they we can easily replace it when we have a better for whole site.  Mostly we rely on status of MobxPromise, so this isn't an issue.